48.ArithmeticUnitTest/main.cpp

```diff
@@ -450,7 +450,7 @@ class ArythmeticUnitTestApp : public NonGraphicalApplicationBase
 		core::smart_refctd_ptr<IGPUCommandBuffer> cmdbuf;
 		logicalDevice->createCommandBuffers(cmdPools[0].get(), IGPUCommandBuffer::EL_PRIMARY, 1u, &cmdbuf);
 		computeQueue->startCapture();
-		for (uint32_t workgroupSize=445; workgroupSize<=446; workgroupSize+=100)
+		for (uint32_t workgroupSize=512; workgroupSize<=513; workgroupSize+=100)
```

Comment:
final version needs to test every multiple of max subgroup size

Reply:
actually item count, we'll just round up the workgroup size to next multiple of max subgroup size